fix(10024): eslint generation (#10118)
biomejs/biome: .changeset/weak-kiwis-post.md

---
"@biomejs/biome": patch
---

> Fixed [#10024](https://github.com/biomejs/biome/issues/10024): `biome migrate eslint` now correctly maps a single Eslint rule to multiple Biome rules (if applicable).

Reviewer: It's not very clear what's the impact of this change to the users. See if this suggestions works. I removed the word "map", and used "migrates" instead. 

```suggestion
Fixed [#10024](https://github.com/biomejs/biome/issues/10024): `biome migrate eslint` correctly migrates `eslint` rules that belong to multiple Biome rules.
```

Maybe we should add an example.
